applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_linear_elastic_plane_strain_2D_law.cpp

```diff
@@ -0,0 +1,201 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "boost/numeric/ublas/assignment.hpp"
```

Comment:
Let's put this include in angular brackets to indicate that it is a "system" header rather than a project header.
```suggestion
#include <boost/numeric/ublas/assignment.hpp>
```

Reply:
Done